db/migrate/20241118103032_fulfill_adjusted_fee_unit_precise_amount_cents.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class FulfillAdjustedFeeUnitPreciseAmountCents < ActiveRecord::Migration[7.1]
+  class AdjustedFee < ApplicationRecord
+    belongs_to :invoice
+  end
+
+  class Invoice < ApplicationRecord
+    enum status: {draft: 0}
+  end
+
+  def change
```

Comment:
Does this migration should work the same when we run it up and down(rollback)?

Reply:
oops, yes forgot about this